src/lib/containers/download_list_v2/DownloadCartPage.tsx

```diff
@@ -115,18 +118,96 @@ export const DownloadCartPage:React.FunctionComponent<DownloadCartPageProps> = (
+                      placement="bottom"
+                    >
+                      <IconSvg options={{icon:'info'}} />
+                    </MarkdownPopover>
+                  </div>
+                  <Typography variant={'body1'}>
+                    <ul>
+                      <li>
+                        Eligible files will be added to .Zip packages of up to 1GB in size
+                      </li>
+                      <li>
+                        If you have more than 1GB, you can create multiple packages
+                      </li>
+                      <li>
+                        Will only include files which are hosted on Synapse native storage
+                      </li>
+                      <li>
+                        Packages include a CSV Manifest containing Metadata for each file
+                      </li>
+                    </ul>
+                  </Typography>
+                  <span>
+                    {data.numberOfFilesAvailableForDownloadAndEligibleForPackaging > 0 && <a className="highlight-link" onClick={() => {setIsShowingCreatePackageUI(true)}}>Download As .Zip Packages</a>}
+                    {data.numberOfFilesAvailableForDownloadAndEligibleForPackaging == 0 &&
+                      <a className="highlight-link disabled"
+                      data-for="downloadZipPackagesUnavailable"
+                      data-tip="You cannot create a .zip package<br />because there are no eligible files.">
+                        <ReactTooltip
+                          delayShow={300}
+                          multiline={true}
+                          place="top"
+                          type="dark"
+                          effect="solid"
+                          id="downloadZipPackagesUnavailable"
+                        />
+                        Download As .Zip Packages
+                      </a>}
+                  </span>
+                </div>
+                <div>
+                  <div className="headlineWithHelp">
+                    <Typography variant={'headline3'}>
+                      <IconSvg options={{ icon: 'code' }} /> Programmatic Packages
+                    </Typography>
+                    <MarkdownPopover
+                      contentProps={{ markdown: 'This will provide syntax which you can enter into your programmatic client. It is suitable for large files (>100 MB), for packages > 1GB, and for files which aren’t stored on Synapse native storage (eg. in a special AWS S3 bucket or Google Cloud.  External links will require navigation to an external site, which may require a seperate login process.' }}
```

Comment:
```suggestion
                      contentProps={{ markdown: 'This will provide syntax which you can enter into your programmatic client. It is suitable for large files (>100 MB), for packages > 1GB, and for files which aren’t stored on Synapse native storage (e.g. in a special AWS S3 or Google Cloud bucket).  External links will require navigation to an external site, which may require a separate login process.' }}
```